torch/csrc/autograd/variable.cpp

```diff
@@ -59,6 +66,14 @@ DifferentiableViewMeta::DifferentiableViewMeta(
   }
 }
 
+void DifferentiableViewMeta::set_creation_meta(CreationMeta new_creation_meta) {
+  TORCH_CHECK(
+      has_bw_view(), "creation_meta can only exist for backward views.");
+  creation_meta_ = new_creation_meta;
+  creation_traceback_ = torch::CapturedTraceback::gather(
```

Comment:
Should we check for anomaly mode in this case as well?

Reply:
yes